Flow.Launcher.Infrastructure/StringMatcher.cs

```diff
@@ -43,21 +43,23 @@ public MatchResult FuzzyMatch(string query, string stringToCompare)
         /// </summary>
```

Comment:
please update the FuzzyMatch summary to include the Acronym search steps

Reply:
👌, and I have a question, should we add an option to open acronym search?

Reply:
sorry what do you mean open acronym search?

Reply:
I mean we can have a parameters that indicates whether the search should include acronym search (maybe default is true).

Reply:
any reason why someone wouldnt use it or have the search method included?

Reply:
Emmm I am not quite sure. For example, maybe we don't want acronym search used to matching for explorer plugin because the index search doesn't include acronym search.
Maybe for some description matching (generally in subtitle matching), we don't want acronym search.

Reply:
Good point, though Explorer's search can't utilize fuzzy search at all due to usage of DirectiryInfoSearch and Index, so no need to worry here. 

If we don't want description search might as well not use fuzzy on it either.

Reply:
Well, I mean the Explorer utilize fuzzy search to create highlight data for results in result manager, so that may be a place that shouldn't use acronym search. ```

```

Reply:
but you can use acronym search with Explorer, so you cant type 'gk' and hit GitKraken, you have to type 'git' so highlight data would still be correct right?

Reply:
Yeah I agree. I think there is some conner case but it is not a lot.